src/app/components/navbar/navbar.component.html

```diff
@@ -136,5 +137,70 @@
         </div>
       </li>
     </ul>
+=======
```

Comment:
@CryceTruly this line as well I think we don't need it.

Reply:
thanks, @alexxsanya, I have fixed this